easybuild/easyconfigs/t/TINKER/TINKER-8.7.2-foss-2019b.eb

```diff
@@ -0,0 +1,21 @@
+name = 'TINKER'
+version = '8.7.2'
+
+homepage = 'https://dasher.wustl.edu/tinker'
+description = """The TINKER molecular modeling software is a complete and general package for molecular mechanics
+ and dynamics, with some special features for biopolymers."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'openmp': True}
```

Comment:
doesn't seem to be necessary, the tests are using threads even if built without this (and even this is set to `False`, so it can actually be misleading?)

Reply:
@akesandgren can you address this comment?

Reply:
Yeah, I dropped it.